### ISSUE TITLE
Using StandardNotes on a Single Device.  Conflicted Copy.  How??

### ISSUE THREAD

**Issue by DiagonalArg** [standardnotes/forum#2299]
iPhone SE, gen.1, iOS 15.7.1
StandardNotes v. 3.46.28 (355)

I am *only* using standardnotes on my iphone.  I am confused how it's possible to end up with a "conflicted copy" of a note, when I'm not syncing to another device.

I don't see any way to get you a debug log, or I would have sent it.

**Comment by effieeee** [comment]
Hi @DiagonalArg, it's pretty rare, but that would possibly mean that you had a number of changes on your device which the server was unable to sync in proper succession 🤔 Does this happen fairly often on your device, and have you noticed if it happens during times when internet connectivity isn't strong?

**Comment by DiagonalArg** [comment]
Yes, I agree it's rare, but it seems to me that if there's only one device then it shouldn't ever happen.  

So no, it doesn't happen fairly often.  As far as connectivity goes, I couldn't say.

**Comment by DiagonalArg** [comment]
I guess it's not totally rare, as it just happened to me again.

No way to get you debug logs?

**Comment by effieeee** [comment]
Hmm, there is a way to collect device logs via Xcode, though I'm not yet certain what we will be able to get from it especially since the issue doesn't result in a critical error or crash:

https://www.ibm.com/support/pages/how-collect-ios-logs
